test/Engine/ProtoTest/Associative/MethodResolution.cs

```diff
@@ -313,5 +313,57 @@ public void TestMethodResultionForNonExistType()
             thisTest.VerifyRunScriptSource(code, error);
             thisTest.Verify("z", null);
         }
+

```

Comment:
I think this test is actually a legacy addition that shouldn't have been folded into this PR.